terraform/testcases/kafka_otlp_trace_3_2_0/README.md

```diff
@@ -0,0 +1 @@
+For more details of how this test works, please look into docs/kafka-tests.md
```

Comment:
Will this doc be included in this PR or a subsequent one?

Reply:
I'm good with merging to dev without the doc existing. But it should be a blocker for merging to `terraform`.

Reply:
No, it was my miss not to commit this. I added.